### PR TITLE
Discover k8s generic net policy from kubearmor logs

### PR DESCRIPTION
**Pull request by seswarrajan**
Signed-off-by: Eswar Rajan Subramanian <eswar@accuknox.com>

### CODE REVIEW

File: src/plugin/k8sNetwork.go [hunk 1]
```diff
@@ -0,0 +1,102 @@
...
+
+				k8NetPol.Spec.Ingress = append(k8NetPol.Spec.Ingress, ingressRule)
+			}
+			k8NetPol.Spec.PolicyTypes = append(k8NetPol.Spec.PolicyTypes, nv1.PolicyType(nv1.PolicyTypeIngress))
+		}
+
+		res = append(res, k8NetPol)
+	}
+
+	fmt.Printf("ERS ==> res: %v\n", res)
```
Comment (nyrahul) [inline src/plugin/k8sNetwork.go]: use log() instead.

**Review by nyrahul — APPROVED** [review]
LGTM 👍 Great work! Just one minor comment.

File: src/networkpolicy/helperFunctions.go [hunk 2]
```diff
@@ -764,22 +774,42 @@ func WriteNetworkPoliciesToFile(cluster, namespace string) {
...
+		policies := plugin.ConvertKnoxNetPolicyToK8sNetworkPolicy(cluster, namespace)
+
+		for i := range policies {
+			genericNetPol := wpb.Policy{}
 
-		response.Ciliumpolicy = append(response.Ciliumpolicy, &ciliumpolicy)
+			val, err := json.Marshal(&policies[i])
+			if err != nil {
+				log.Error().Msg(err.Error())
+			}
```
Comment (nyrahul) [inline src/networkpolicy/helperFunctions.go]:
```suggestion
			if err != nil {
				log.Error().Msg(err.Error())
				continue
			}
```

**Review by Ankurk99 — APPROVED** [review]
Just some lint errors, otherwise looks good

**Review by nyrahul — APPROVED** [review]

**Comment by Ankurk99** [comment]
@seswarrajan Can you please squash the commits? Then let's merge this :+1:

**Pull request merged**